scripts/bb-test-zfstests.sh

```diff
@@ -18,17 +18,30 @@ if echo "$TEST_ZFSTESTS_SKIP" | grep -Eiq "^yes$|^on$|^true$|^1$"; then
 fi
 
 CONSOLE_LOG="$PWD/console.log"
+SUMMARY_LOG="$PWD/summary.log"
+TEST_LOG="$PWD/test.log"
 KMEMLEAK_LOG="$PWD/kmemleak.log"
 KMEMLEAK_FILE="/sys/kernel/debug/kmemleak"
 RESULT=0
 
 # Cleanup the pool and restore any modified system state.  The console log
 # is dumped twice to maximize the odds of preserving debug information.
+# Generate a summary of results and place them in a different file.
 cleanup()
 {
     dmesg >$CONSOLE_LOG
     sudo -E $ZFS_SH -u
     dmesg >$CONSOLE_LOG
+
+    if [ -f "$TEST_LOG" ]; then
+        grep -A 20 "Results Summary" "$TEST_LOG" > $SUMMARY_LOG
+        echo "" >> $SUMMARY_LOG
+        grep -a "\[KILLED\]" log >> $SUMMARY_LOG
+        grep -a "\[FAIL\]" log >> $SUMMARY_LOG
+        echo "" >> $SUMMARY_LOG
+        awk '/\[FAIL\]|\[KILLED\]/{ show=1; print; next; } ' \
```

Comment:
Looks like the `awk` needs a little tweaking.

`awk: fatal: cannot open file `/\[SKIP\]|\[PASS\]/{ show=0; } show' for reading (No such file or directory)`

http://build-dev.zfsonlinux.org/builders/Fedora%2027%20x86_64%20%28TEST%29/builds/3